ql/src/java/org/apache/hadoop/hive/ql/exec/ReplCopyTask.java

```diff
@@ -61,7 +67,10 @@ protected int execute(DriverContext driverContext) {
     LOG.debug("ReplCopyTask.execute()");
     FileSystem dstFs = null;
     Path toPath = null;
+
     try {
+      Hive hiveDb = Hive.get();
```

Comment:
Should we use getHive() of Task class to consider the Task conf?

Reply:
done